Userland/Libraries/LibWeb/DOM/Element.h

```diff
@@ -372,6 +373,8 @@ class Element
 
     Optional<FlyString> const& id() const { return m_id; }
 
+    virtual HTML::LazyLoadingElement& dom_element_to_lazy_loading_element() { VERIFY_NOT_REACHED(); }
```

Comment:
We could return a pointer here to move the assertion into the caller of the method, rather than having a virtual that always asserts? Not sure it makes any semantic difference to the code, but it might look cleaner.